### ISSUE TITLE
Install Guide

### ISSUE THREAD

**Issue by Stormeal** [Qwiso/QMultiMod#5]
Hey, could you provide us with an install guide in the readme? Having some trouble installing the mod.

**Comment by Vlad-00003** [comment]
There is no installation guide at all... Just grab the latest version [nexus page](https://www.nexusmods.com/subnautica/mods/33) and put it in the qmods folder created by qmodloader.

If there is any problem with it - check if QModLoader is installed correctly.

**Comment by Stormeal** [comment]
Thanks a million! 👍

**Comment by ghost** [comment]
I can't get the mod to work. I have reinstalled QMods and every other mod works. Any tips?

**Comment by Vlad-00003** [comment]
Over mods works fine? Can you post your output_log.txt somewhere, on http://pastebin.com, for example, and leave the link here, so we can see?

**Comment by ghost** [comment]
Yes, every mod works fine but the game doesn't seem to load this one. I've tried using only this mod too but it still doesn't load.

https://pastebin.com/mY1KrsvC

**Comment by ghost** [comment]
I tried starting the game with only this mod again and checked the output_log, it appears to be an issue with "FireExtinguisherHolderRechargeValue" so I removed it and now it is working. I feel so stupid, thanks for the help!

**Comment by Vlad-00003** [comment]
what? Once again - what the problem was in the first place?

**Comment by ghost** [comment]
The problem was that the mod didn't even load when I was in the game. There was no difference on anything. The game just ran as vanilla but all the other mods worked fine. There was no crash or anything, it just didn't work in the game.

**Comment by Vlad-00003** [comment]
and you've fixed it... how? Just by removing 1 option from the config? O_o

**Comment by ghost** [comment]
Yes, I checked the output_log when I only had this mod active and there was an error message about that, so I removed the line and now it works.

But now I put that line back in and it is still working... It must have just been a coincidence and I have now idea what I've done. Wtf...

**Comment by Vlad-00003** [comment]
This is the coding, baby ( ͡° ͜ʖ ͡°)

**Comment by raymerjacque** [comment]
Install guide is not on nexusmod anymore, link is dead. please how to install this mod ?
